src/Akeneo/Tool/Bundle/FileStorageBundle/spec/Validator/Constraints/UploadedFileValidatorSpec.php

```diff
@@ -13,17 +13,17 @@
 
 final class UploadedFileValidatorSpec extends ObjectBehavior
 {
-    public function let(ExecutionContextInterface $context)
+    public function let(ExecutionContextInterface $context): void
     {
         $this->initialize($context);
     }
 
-    function it_is_initializable()
+    public function it_is_initializable(): void
     {
         $this->shouldHaveType(Constraints\UploadedFileValidator::class);
     }
 
-    public function it_validates_the_file(
+    public function it_validates_a_correct_file(
         File\UploadedFile $file,
         ExecutionContextInterface $context
     ) {
```

Comment:
```suggestion
    ): void {
```
Add return type to be consistent with other functions